ReleaseHistory.md

```diff
@@ -1,4 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+## **v4.5.1 UNRELEASED
+* BUG: Report `WRN997_InvalidTarget` if the path of the scan target contains invalid characters.
```

Comment:
>Report [](http://example.com/codeflow?start=7&length=6)

this doesn't describe the symptoms of the original string.

Reply:
Updated with symptom.